prebuilt/common/etc/init.d/30permissions

```diff
@@ -0,0 +1,5 @@
+#!/system/bin/sh
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit